cypress/integration/equipments.spec.ts

```diff
@@ -0,0 +1,72 @@
+beforeEach(() => {
+  cy.initializeSession('UserOfficer');
+
+  cy.visit({
+    url: '/equipments',
+    timeout: 15000,
+  });
+});
+
+describe('Add equipment', () => {
+  it('should be able to add a new equipment', () => {
+    cy.get('[data-cy=btn-new-equipment]').click();
+
+    cy.get('[data-cy=name]').type('1 Test equipment');
+    cy.get('[data-cy=btn-save-equipment]').click();
+
+    cy.contains(/1 Test equipment/i);
+    cy.contains(/Anders Andersson/i);
+    cy.contains(/Has no scheduled maintenance/i);
+  });
+});
+
+describe('Edit equipment', () => {
+  beforeEach(() => {
+    cy.get('[data-cy=btn-view-equipment]')
+      .first()
+      .click();
+
+    cy.contains(/1 Test equipment/i);
+  });
+
+  it('should be able to turn on auto accept', () => {
+    cy.get('[data-cy=btn-edit-equipment]').click();
+    cy.get('[data-cy=autoAccept]').click();
+
+    cy.get('[data-cy=btn-save-equipment]').click();
+
+    cy.get('[data-cy=autoAccept]').contains(/yes/i);
+  });
+
+  it('should be able to set under maintenance indefinitely', () => {
+    cy.get('[data-cy=btn-edit-equipment]').click();
+    cy.get('[data-cy=underMaintenance]').click();
+
+    cy.get('[data-cy=btn-save-equipment]').click();
+
+    cy.contains(/Under maintenance indefinitely/i);
+  });
+
+  it('should be able to turn off maintenance', () => {
+    cy.get('[data-cy=btn-edit-equipment]').click();
+    cy.get('[data-cy=underMaintenance]').click();
+
+    cy.get('[data-cy=btn-save-equipment]').click();
+
+    cy.contains(/Has no scheduled maintenance/i);
+  });
+
+  it('should be able to set under maintenance for a specified interval', () => {
+    cy.get('[data-cy=btn-edit-equipment]').click();
+    cy.get('[data-cy=underMaintenance]').click();
+    cy.get('[data-cy=maintenanceTime-defined]').click();
+
+    cy.get('[data-cy=maintenanceStartsAt] input').clear();
+    cy.get('[data-cy=maintenanceStartsAt]').type('2020-09-01 08:00');
+    cy.get('[data-cy=maintenanceEndsAt]').type('2020-10-01 08:00');
```

Comment:
Not sure if it is included in the scheduler-frontend but maybe it is a good idea to use faker for generating some random dates and inputs.